spec/unit/module_spec.rb

```diff
@@ -90,14 +90,18 @@
         }.not_to raise_error
       end
       describe 'the exclude_spec setting' do
+        it 'sets the exclude_spec instance variable to false by default' do
+          obj = R10K::Module.new(name, '/modulepath', options)
+          expect(obj.instance_variable_get("@exclude_spec")).to eq(false)
+        end
         it 'sets the exclude_spec instance variable' do
           obj = R10K::Module.new(name, '/modulepath', options.merge({exclude_spec: true}))
           expect(obj.instance_variable_get("@exclude_spec")).to eq(true)
         end
-        it 'can be overridden by the overrides map' do
+        it 'cannot be overridden by the overrides map' do
```

Comment:
Can we clarify where things in the overrides map come from in this test description? I definitely lose track and it would be nice to have a little more documentation of the internals here.

Reply:
I pushed a spec update to be clearer and revamped the configuration doc to be explicit about the setting behavior and precedence.